src/softphone.js

```diff
@@ -192,6 +192,8 @@
 
                     session.remoteAudioElement = document.getElementById('remote-audio');
                     session.connect();
+                    var bus = connect.core.getEventBus();
+                    bus.trigger(contact.getEventName(connect.ContactEvents.SESSION), session);
```

Comment:
also should this go inside - session.onSessionConnected  callback to make sure we have the active RTC session to work with??